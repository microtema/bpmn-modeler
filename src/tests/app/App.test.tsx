import * as React from 'react';
import * as ReactDOM from 'react-dom';
import App from '../../app/App';

describe('App', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('render correctly text component', () => {
    expect({ foo: 'bar' }).toMatchSnapshot();
  });
});
