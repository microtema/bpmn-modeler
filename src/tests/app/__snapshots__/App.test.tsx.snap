// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render correctly text component 1`] = `
Object {
  "foo": "bar",
}
`;
